MetaMask/metamask-extension
# fix(000): style improvement for security provider part in settings

## **Description**

There are some styles mismatching in new security session in settings tab.

## **Related issues**

Fixes:  N/A

## **Manual testing steps**

1. Go to settings
2. Check experimental tab
3. validate styles in "select your providers"

## **Screenshots/Recordings**



### **Before**
<img width="1000" alt="Screenshot 2023-11-01 at 01 59 17" src="https://github.com/MetaMask/metamask-extension/assets/12678455/bde46819-e411-4217-a0d4-235d558539d8">



### **After**

<img width="1003" alt="Screenshot 2023-11-01 at 02 35 52" src="https://github.com/MetaMask/metamask-extension/assets/12678455/cdb01c19-a28a-4660-8766-ef0c4f89c58e">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
